src/vm_pages.rs

```diff
@@ -1262,6 +1318,151 @@ impl<'a, T: GuestStagePagingMode> FinalizedVmPages<'a, T> {
         self.do_remove_region(page_addr, len, VmRegionType::Mmio)
     }
 
+    /// Converts the specified memory region from confidential to shared.
+    pub fn share_mem_region(&self, page_addr: GuestPageAddr, len: u64) -> Result<()> {
+        self.do_share_mem_region(page_addr, len, true)
+    }
+
+    /// Converts the specified memory region from shared to confidential.
+    pub fn unshare_mem_region(&self, page_addr: GuestPageAddr, len: u64) -> Result<()> {
+        self.do_share_mem_region(page_addr, len, false)
+    }
+
+    pub fn do_share_mem_region(
```

Comment:
nit: maybe `do_convert_*` for these helpers instead of `do_share_*`? though i guess "convert" is overloaded...

Reply:
Yes that's fine I can use `do_convert_region_*`